patches/com/mojang/blaze3d/opengl/GlConst.java.patch

```diff
@@ -1,31 +1,266 @@
 --- a/com/mojang/blaze3d/opengl/GlConst.java
 +++ b/com/mojang/blaze3d/opengl/GlConst.java
-@@ -251,6 +_,8 @@
-             case RED8 -> 33321;
-             case RED8I -> 33329;
-             case DEPTH32 -> 33191;
-+            case DEPTH24_STENCIL8 -> org.lwjgl.opengl.GL32.GL_DEPTH24_STENCIL8;
-+            case DEPTH32_STENCIL8 -> org.lwjgl.opengl.GL32.GL_DEPTH32F_STENCIL8;
+@@ -147,14 +_,18 @@
+             case EQUAL_DEPTH_TEST -> 514;
+             case LESS_DEPTH_TEST -> 513;
+             case GREATER_DEPTH_TEST -> 516;
+-            default -> 515;
++            case LEQUAL_DEPTH_TEST -> 515;
++            case NOT_EQUAL_DEPTH_TEST -> org.lwjgl.opengl.GL11C.GL_NOTEQUAL;
++            case GEQUAL_DEPTH_TEST -> org.lwjgl.opengl.GL11C.GL_GEQUAL;
++            case NEVER_DEPTH_TEST -> org.lwjgl.opengl.GL11C.GL_NEVER;
          };
      }
  
-@@ -260,6 +_,10 @@
-             case RED8 -> 6403;
-             case RED8I -> 6403;
-             case DEPTH32 -> 6402;
-+            // As long as null data will be passed to texImage2D, the external format does not matter as long as it matches the internalFormat.
-+            // So the usage, for depth, of unsigned int in one case and float in the other case, is not a problem.
-+            case DEPTH24_STENCIL8 -> org.lwjgl.opengl.GL32.GL_DEPTH_STENCIL;
-+            case DEPTH32_STENCIL8 -> org.lwjgl.opengl.GL32.GL_DEPTH_STENCIL;
+     public static int toGl(PolygonMode p_412150_) {
+         return switch (p_412150_) {
+             case WIREFRAME -> 6913;
+-            default -> 6914;
++            case FILL -> 6914;
++            case POINT -> org.lwjgl.opengl.GL11C.GL_POINT;
          };
      }
  
-@@ -269,6 +_,8 @@
-             case RED8 -> 5121;
-             case RED8I -> 5121;
-             case DEPTH32 -> 5126;
-+            case DEPTH24_STENCIL8 -> org.lwjgl.opengl.GL32.GL_UNSIGNED_INT_24_8;
-+            case DEPTH32_STENCIL8 -> org.lwjgl.opengl.GL32.GL_FLOAT_32_UNSIGNED_INT_24_8_REV;
+@@ -174,6 +_,11 @@
+             case SRC_ALPHA -> 770;
+             case SRC_COLOR -> 768;
+             case ZERO -> 0;
++            case SRC_ALPHA_SATURATE -> org.lwjgl.opengl.GL11C.GL_SRC_ALPHA_SATURATE;
++            case SRC1_COLOR -> org.lwjgl.opengl.GL33C.GL_SRC1_COLOR;
++            case ONE_MINUS_SRC1_COLOR -> org.lwjgl.opengl.GL33C.GL_ONE_MINUS_SRC1_COLOR;
++            case SRC1_ALPHA -> org.lwjgl.opengl.GL15C.GL_SRC1_ALPHA;
++            case ONE_MINUS_SRC1_ALPHA -> org.lwjgl.opengl.GL33C.GL_ONE_MINUS_SRC1_ALPHA;
```

Comment:
What about the GL33 ones here?

Reply:
This probably slipped through because GL3xC.

Requirement here is GL 3.3 or GL_ARB_blend_func_extended
Can add a comment about the requirement to the enum definition, and later will also need to add a capability check.